tests/Microsoft.Identity.Test.Unit.net45/CoreTests/WsTrustTests/WsTrustBuilderTests.cs

```diff
@@ -25,26 +25,26 @@
 ////
 ////------------------------------------------------------------------------------
 
-//using System;
-//using System.Collections.Generic;
-//using System.Diagnostics;
-//using System.IO;
-//using System.Linq;
-//using System.Net;
-//using System.Net.Http;
-//using System.Text;
-//using System.Threading.Tasks;
-//using System.Xml;
-//using System.Xml.Schema;
-//using Microsoft.Identity.Client;
-//using Microsoft.Identity.Client.Http;
-//using Microsoft.Identity.Client.WsTrust;
-//using Microsoft.VisualStudio.TestTools.UnitTesting;
-//using Test.Microsoft.Identity.Client.Unit;
-//using Test.Microsoft.Identity.Client.Unit.Mocks;
-
-//namespace Test.Microsoft.Identity.Unit.WsTrustTests
-//{
+// using System;
+// using System.Collections.Generic;
+// using System.Diagnostics;
+// using System.IO;
+// using System.Linq;
+// using System.Net;
+// using System.Net.Http;
+// using System.Text;
+// using System.Threading.Tasks;
+// using System.Xml;
+// using System.Xml.Schema;
+// using Microsoft.Identity.Client;
+// using Microsoft.Identity.Client.Http;
+// using Microsoft.Identity.Client.WsTrust;
+// using Microsoft.VisualStudio.TestTools.UnitTesting;
+// using Test.Microsoft.Identity.Client.Unit;
+// using Test.Microsoft.Identity.Client.Unit.Mocks;
+
+// namespace Test.Microsoft.Identity.Unit.WsTrustTests
+// {
 //    [TestClass]
```

Comment:
Hmm, why are these commented out? Is the DeploymentItem attribute failing?

Reply:
Don't know.  They were already commented out.  I can look at the changelog on this file in a bit and see if we need to delete or fix this file.